nat-lab/bin/kill_process_by_natlab_id

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+NATLAB_ID=$1
+
+for pid in $(ps -e -o pid=); do
```

Comment:
How many processes are there typically in a container? This looks like `O(N)` algorithm, I wonder if its a problem or not.

Reply:
Two are run in the beginning and then we start another few - it looks that it should be less than 10